modules/apps/foundation/frontend-image-editor/frontend-image-editor-web/.task-cache/resources/ImageEditor.scss

```diff
@@ -0,0 +1,220 @@
+@import "bourbon";
+@import "mixins";
+
+$imageeditor-btn-padding: 8px 4px !default;
+$imageeditor-btn-space: 24px !default;
+$imageeditor-controls-height: 52px !default;
+$imageeditor-filters-border-width: 4px;
+$imageeditor-filters-height: 90px !default;
+$imageeditor-history-controls-top: 30px;
+$slider-height: 20px;
+
+// Skin
+
+$brand-primary: darken(#428BCA, 6.5%) !default; // #337ab7
+
+$imageeditor-bg: #111314 !default;
+$imageeditor-filters-text-color: #57636A !default;
+$imageeditor-history-controls-btn-color: #4E6371 !default;
+$imageeditor-link-color: $brand-primary !default;
+$imageeditor-text-color: #FFF !default;
+
+$gray-base: #000 !default;
+$gray-light: lighten($gray-base, 46.7%) !default; // #777
+$gray-lighter: lighten($gray-base, 93.5%) !default; // #eee
+
+body {
+	background-color: $imageeditor-bg !important;
```

Comment:
I've realized that after the integration is completed, we have a different place for this rather than `body`, so I'll amend this in a future commit.